ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -0,0 +1,531 @@
+  const moduleAction = registrationTask && (module === AccessModule.TWOFACTORAUTH ?
+      () => setShowTwoFactorAuthModal(true) :
+      registrationTask.onClick);
+
+  const Refresh = () => <Button
+      type='primary'
+      style={styles.refreshButton}
+      onClick={() => location.reload()} >
+    <Repeat style={styles.refreshIcon}/> Refresh
+  </Button>;
+
+  const Next = () => <FlexRow style={styles.nextElement}>
+    <span style={styles.nextText}>NEXT</span> <ArrowRight style={styles.nextIcon}/>
+  </FlexRow>;
+
+  const eligible = true; // TODO RW-7059
+  const ModuleIcon = () => <div style={styles.moduleIcon}>
+    {cond(
+      // not eligible to complete module
+      [!eligible, () => <MinusCircle data-test-id={`module-${module}-ineligible`} style={{color: colors.disabled}}/>],
+      // eligible and (completed or bypassed)
+      [eligible && !!statusTextMaybe, () => <CheckCircle data-test-id={`module-${module}-complete`} style={{color: colors.success}}/>],
+      // eligible and incomplete and unbypassed
+      [eligible && !statusTextMaybe, () => <CheckCircle data-test-id={`module-${module}-incomplete`} style={{color: colors.disabled}}/>])}
+  </div>;
+
+  const ModuleBox = ({children}) => {
+    return active ?
+        <Clickable onClick={() => {
+          setShowRefresh(true);
+          moduleAction();
+        }}>
+          <FlexRow style={styles.activeModuleBox}>{children}</FlexRow>
+        </Clickable> :
+        <FlexRow style={styles.inactiveModuleBox}>{children}</FlexRow>;
+  };
+
+  const Module = () => <FlexRow data-test-id={`module-${module}`}>
+    <FlexRow style={styles.moduleCTA}>
+      {active && (showRefresh ? <Refresh/> : <Next/>)}
+    </FlexRow>
+    <ModuleBox>
+      <ModuleIcon/>
+      <FlexColumn>
+        <div style={active ? styles.activeModuleText : styles.inactiveModuleText}>
+          {moduleLabels.get(module)}
+        </div>
+        {statusTextMaybe && <div style={styles.moduleDate}>{statusTextMaybe}</div>}
+      </FlexColumn>
+    </ModuleBox>
+    {showTwoFactorAuthModal && <TwoFactorAuthModal
+        onClick={() => setShowTwoFactorAuthModal(false)}
+        onCancel={() => setShowTwoFactorAuthModal(false)}/>}
+  </FlexRow>;
+
+  const moduleEnabled = !!registrationTask;
+  return moduleEnabled ? <Module/> : null;
+};
+
+export const DataAccessRequirements = fp.flow(withProfileErrorModal)((spinnerProps: WithSpinnerOverlayProps) => {
+  const {profile, reload} = useStore(profileStore);
+
+  const syncExternalModulesAndReloadProfile = async() => {
+    const aborter = new AbortController();
+    spinnerProps.showSpinner();
+    await Promise.all([
+      profileApi().syncTwoFactorAuthStatus({signal: aborter.signal}),
+      profileApi().syncComplianceTrainingStatus({signal: aborter.signal}),
+      profileApi().syncEraCommonsStatus({signal: aborter.signal}),
+    ]);
+    spinnerProps.hideSpinner();
+    reload();
+
+    // cleanup on unmount
+    return aborter.abort;
```

Comment:
opt: try/catch around the above promise.all for Abort errors could reduce error logs here, since it is expected (detectable using https://github.com/all-of-us/workbench/blob/master/ui/src/app/utils/errors.tsx#L22)